Assets/Source/Components/Environment/ParallaxLayer.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Assets.Source.Components.Base;
+using UnityEngine;
+
+public class ParallaxLayer : ComponentBase
+{
+    public ParallaxGroup ParallaxGroup { get; set; }
```

Comment:
These actually don't show up in the inspector variables because they are auto properties and have private backing fields that can't be accessed by unity (yay)